### PR TITLE
Fix bugs in BatchNormalization layer

### PR DESCRIPTION
**Pull request by caisq**
* Bugs related to the handling of `scale` and `center` config fields.

Fixes: https://github.com/tensorflow/tfjs/issues/43

### CODE REVIEW

**Comment by caisq** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions, all commit checks successful.

---

*[src/layers/normalization_test.ts, line 102 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/104#-L8zRDm88t9yrFU0Tz9S:-L90p4P26JpA04SopIRj:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/40cc62336537eb08854d4cb2b7d0f6ed7d8eadb1/src/layers/normalization_test.ts#L102)):*
<details><summary><i>Previously, dsmilkov (Daniel Smilkov) wrote…</i></summary><blockquote>

typo: no scale
</blockquote></details>

Done.

---

*[src/layers/normalization_test.ts, line 113 at r1](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/104#-L8zRJKK7O3vJWlBlMbB:-L90p4xPBQBwCjT33eSy:ba6vxjl) ([raw file](https://github.com/tensorflow/tfjs-layers/blob/40cc62336537eb08854d4cb2b7d0f6ed7d8eadb1/src/layers/normalization_test.ts#L113)):*
<details><summary><i>Previously, dsmilkov (Daniel Smilkov) wrote…</i></summary><blockquote>

Quick q: this todo depends on gradients of batchnorm to be implemented right? Might be good to add that to the comment.
</blockquote></details>

Good question. BatchNormalization weights updating actually doesn't use gradients. For example,  the moving average and moving variances weights are marked as not trainable:
https://github.com/keras-team/keras/blob/master/keras/layers/normalization.py#L122

The BatchNormalization layer has its own update function:
https://github.com/keras-team/keras/blob/master/keras/layers/normalization.py#L191

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/tensorflow/tfjs-layers/104)*
